ui/src/app/views/homepage/component.html

```diff
@@ -1,181 +1,182 @@
-        <div class="row">
-          <div class="card add-card" (click)="addWorkspace()">
-            <div class="card-text">
-              Create a <br/> New Workspace <br/>
-              <clr-icon shape="plus-circle" class="plus-circle"></clr-icon>
+          <div class="row">
+            <div class="col-md-3" style="margin-bottom: 2rem;">
+              <div class="card add-card" (click)="addWorkspace()">
+                <div class="card-text">
+                  Create a <br/> New Workspace <br/>
+                  <clr-icon shape="plus-circle" class="plus-circle"></clr-icon>
+                </div>
+              </div>
             </div>
+
           </div>
         </div>
       </div>
-
     </div>
-  </div>
-  <div class="footer">
-    <div class="row">
-      <div class="footerTitle">How to Use the All of Us Researcher Workbench</div>
-    </div>
-    <div class="row">
-      <a href="#" class="footerLink" data-toggle="tooltip" title="Coming Soon">
-        See all documentation</a>
-    </div>
-    <div class="row textGroup">
-      <div class="footerText">
-        <div class="title">Working Within Researcher Workbench</div>
-        <clr-tooltip>
-          <ul class="bullets" clrTooltipTrigger>
-            <li>
-              <a href="#" class="bullets">
-                Researcher Workbench Mission</a>
-             </li>
-            <li><a href="#" class="bullets">
-              User interface components</a></li>
-            <li><a href="#" class="bullets">
-              What to do when things go wrong</a></li>
-            <li><a href="#" class="bullets">
-              Contributing to the Workbench</a></li>
-          </ul>
-          <clr-tooltip-content *clrIfOpen>
+    <div class="footer">
+      <div class="row">
+        <div class="footerTitle">How to Use the All of Us Researcher Workbench</div>
+      </div>
+      <div class="row footerRtLink">
+        <clr-tooltip clrPosition="top-right">
+          <a href="#" class="footerLink" clrTooltipTrigger>
+            See all documentation</a>
+          <clr-tooltip-content clrSize="sm" *clrIfOpen>
             <span>Coming Soon</span>
           </clr-tooltip-content>
         </clr-tooltip>
+      </div>
+      <div class="row textGroup">
+        <div class="footerText">
+          <div class="title">Working Within Researcher Workbench</div>
+          <clr-tooltip>
+            <ul class="bullets" clrTooltipTrigger>
+              <li>
+                <a href="#" class="bullets">
+                  Researcher Workbench Mission</a>
+              </li>
+              <li><a href="#" class="bullets">
+                User interface components</a></li>
+              <li><a href="#" class="bullets">
+                What to do when things go wrong</a></li>
+              <li><a href="#" class="bullets">
+                Contributing to the Workbench</a></li>
+            </ul>
+            <clr-tooltip-content clrPosition="bottom-left" clrSize="sm" *clrIfOpen>
```

Comment:
Did you check in with Lou on where to put the tooltips? I personally would probably put them above the group, rather than below, because its less likely to not be visible.